bigquery-antipattern-recognition/src/main/java/com/google/zetasql/toolkit/antipattern/Main.java

```diff
@@ -56,70 +35,57 @@ public class Main {
 
   private static final Logger logger = LoggerFactory.getLogger(Main.class);
   private static AntiPatternCommandParser cmdParser;
-  private static String analyzerProject = null;
-  private static AnalyzerOptions analyzerOptions;
-  private static BigQueryAPIResourceProvider resourceProvider;
-  private static ZetaSQLToolkitAnalyzer analyzer;
-  private static BigQueryService service;
-  private static BigQueryCatalog catalog;
-  private static LanguageOptions languageOptions = new LanguageOptions();
-  private static HashMap<String, Integer> visitorMetricsMap;
   private static int countQueriesRead = 0;
   private static int countQueriesWithAntipattern = 0;
 
-  static {
-    languageOptions.enableMaximumLanguageFeatures();
-    languageOptions.setSupportsAllStatementKinds();
-    languageOptions.enableReservableKeyword("QUALIFY");
-  }
 
   public static void main(String[] args) throws ParseException, IOException {
     cmdParser = new AntiPatternCommandParser(args);
-    if (cmdParser.useAnalyzer()) {
-      setAnalyzerOptions();
-    }
+
+    LanguageOptions languageOptions = new LanguageOptions();
+    languageOptions.enableMaximumLanguageFeatures();
+    languageOptions.setSupportsAllStatementKinds();
+    languageOptions.enableReservableKeyword("QUALIFY");
+    AntiPatternHelper antiPatternHelper = new AntiPatternHelper(cmdParser.getProcessingProject(), cmdParser.useAnalyzer(), languageOptions);
```

Comment:
Can we also move the language options to the AntiPatternHelper?

Reply:
Done